src/plugins/unified_search/public/query_string_input/query_string_input.tsx

```diff
@@ -22,7 +22,6 @@ import {
   EuiTextArea,
   htmlIdGenerator,
   PopoverAnchorPosition,
-  toSentenceCase,
```

Comment:
My dear Lukas, I am always fond of cleanups but how is this change related to the push flyout in Discover?

Reply:
Good question :) Up a little bit @andreadelrio had pointed out that sometimes the resizable flyout can cause the placeholder text in the search bar to wrap, and suggested we update it:

> - Design detail for this scenario: Adjust the KQL placeholder text so it doesn't get cut off (to `Filter your data`).

Happy to hear your input here as well!

Reply:
I see, @andreadelrio the reason tha Caroline and I added the language to the placeholder is because there is no indication in the UI about which language is on. To see that a user should open the menu popover (but it is not obvious

<img width="705" alt="image" src="https://github.com/elastic/kibana/assets/17003240/3d2c65a0-8dbf-4a2c-bd5d-791d45db3c04">

So I propose:
- Change to a shorter text which included the language
- Don't do this change now, but take this under consideration to your unified search redesign

Reply:
@lukasolson I was hoping we would only shorten the text when the screen wasn't wide enough to display the whole string. In any case, I hadn't considered the context @stratoula is mentioning (thanks!). I suggest we hold off on this change until we implement other changes to the unified search bar. We're considering exposing the language toggle in the main bar.

Reply:
Yes this sounds better! Thanx Andrea and Lukas!

Reply:
:+1: Reverted in dfebc4061805a9874ac219255b5df1dae9ac2de9